packages/sdk/src/Segments.ts

```diff
@@ -7,19 +9,28 @@ export class Segments {
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#create-segment
 	 */
-	public create(segment: $FixMe) {
-		return this.client.post('/segments', segment)
+	public create(segment: T.SegmentsCreate) {
+		return this.client.post<T.SegmentsCreateResponse>('/segments', segment)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#get-segment
 	 */
 	public get(segmentId: string) {
-		return this.client.get(`/segments/${encode(segmentId)}`)
+		return this.client.get<T.SegmentsGetResponse>(`/segments/${encode(segmentId)}`)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#delete-segment
 	 */
 	public delete(segmentId: string) {
 		return this.client.delete(`/segments/${encode(segmentId)}`)
 	}
+
+	// I see that list method is missing
+
+	/**
+	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#list-segments
+	 */
+	public list(customerId: string) {
+		return this.client.delete(`/customers/${encode(customerId)}/segments`)
```

Comment:
Why `list` method is doing `this.client.delete`? It's missing definition for return type

Reply:
My bad, sorry.